database/migrations/2024_03_23_064740_create_ads_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('ads', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->text('content');
+            $table->longText('cover');
```

Comment:
what's "cover" ? Or you could use other term for this column

Reply:
We will use this column for images.
I will change this to "image".

We are going to use these images for HomePage cover slides, so we meant this title.